### ISSUE TITLE
Years before 1900 or after 2099 not permitted when entering rights metadata through the AM dashboard

### ISSUE THREAD

**Issue by jfcarrano** [archivematica/Issues#1674]
**Expected behaviour**
Entering years to the rights metadata should allow for any 4 digit year to be entered

**Current behaviour**
Only years 1900-2099 and before can be entered

**Steps to reproduce**
Start a transfer in the AM dashboard. Click the notepad to enter metadata. Under rights click "Add". Attempt to enter a year before 1900 or after 2099 and no digit will appear.

**Your environment (version of Archivematica, operating system, other relevant details)**
Archivematica 13.2

---

**For Artefactual use:**

Before you close this issue, you must check off the following:

- [ ] All pull requests related to this issue are properly linked
- [ ] All pull requests related to this issue have been merged
- [ ] A testing plan for this issue has been implemented and passed (testing plan information should be included in the issue body or comments)
- [ ] Documentation regarding this issue has been written and merged (if applicable)
- [ ] Details about this issue have been added to the release notes (if applicable)

**Comment by replaceafill** [comment]
@jfcarrano Would you consider this a duplicate of https://github.com/archivematica/Issues/issues/348?

**Comment by jfcarrano** [comment]
@replaceafill makes sense! I missed that when searching. The 22nd century and beyond dates issue could be called out specifically in that issue to make it clear.
